javaslang/src/main/java/javaslang/collection/Iterator.java

```diff
@@ -1666,7 +1666,7 @@ public U getNext() {
     @Override
     default Iterator<Seq<T>> sliding(long size, long step) {
         if (size <= 0 || step <= 0) {
-            throw new IllegalArgumentException(String.format("size: %s or step: %s not positive", size, step));
+            throw new IllegalArgumentException("size: " + size + " or step: " + step + " not positive");
```

Comment:
In my prototype of javaslang-gwt from javaslang excluded concurrent package, conversion to/from java.util.Stream and reflection. It work :)
Now I'm waiting for java.util.Stream support in GWT (a few days ago Spliterators has been added) and will make next attempt :)